src/bilder/components/hashicorp/vault/models.py

```diff
@@ -67,12 +70,17 @@ class VaultAgentCache(FlexibleBaseModel):
     use_auto_auth_token: Union[str, bool] = True
 
 
-class VaultSealConfig(FlexibleBaseModel, abc.ABC):
-    pass  # noqa: WPS420, WPS604
+class VaultAwsKmsSealConfig(FlexibleBaseModel):
```

Comment:
What does the `FlexibleBaseModel` provide us? I see the class definition, but not sure I understand what it's helping us with

Reply:
It lets us add additional, arbitrary attributes to the class object without it throwing an exception. The intent being that we can add configuration parameters for classes that aren't completely fleshed out to match what the project is expecting for configuration inputs.